plugins/node/opentelemetry-instrumentation-pino/src/instrumentation.ts

```diff
@@ -135,7 +139,7 @@ export class PinoInstrumentation extends InstrumentationBase {
     };
   }
 
-  private _combineOptions(options?: pino.LoggerOptions) {
+  private _combineOptions(options?: any) {
```

Comment:
You could actually keep the `pino.LoggerOptions` type here as it is a private function and a .d.ts file includes only the property name of private members but no types.

Reply:
That would invite someone to add types to public functions, I think, as they are already imported. I would do that, at least, if there was no comment. :)

Reply:
Yeah. A test to verify that types are not part of the exports in the .d.ts files would be nice. But I have no idea how this could be done.